lesson_2/lesson_2_6.py

```diff
@@ -0,0 +1,9 @@
+# Напишите программу используя цикл с предусловием while для вывода каждого четного положительного числа от 0 до 20
+
+if __name__ == '__main__':
+    i = 0
+
+    while i <= 20:
+        if i % 2 == 0 and i > 1:
```

Comment:
і постійно зростає і починається з 0, тому відємним бути не може
```suggestion
        if i % 2 == 0:
```